comms/dht/src/config.rs

```diff
@@ -155,8 +155,8 @@ impl Default for DhtConfig {
             saf_auto_request: true,
             saf_max_message_size: 512 * 1024,
             saf_minimum_request_period: Duration::from_secs(3 * 24 * 60 * 60), // 3 days
-            msg_hash_cache_capacity: 100_000,
-            msg_hash_cache_ttl: Duration::from_secs(5 * 60),
+            dedup_cache_capacity: 2_500,
+            dedup_cache_trim_period: Duration::from_secs(5 * 60),
```

Comment:
nit: use interval in the name to indicate that it's an interval between repeating
```suggestion
            dedup_cache_trim_interval: Duration::from_secs(5 * 60),
```